docs/Configuration.md

```diff
@@ -147,3 +147,70 @@ they get annotated with the reason and comment from the resolution.
 
 For details about the available reasons, see
 [ErrorResolutionReason.kt](../model/src/main/kotlin/config/ErrorResolutionReason.kt).
+
+## Global configuration
+
+This section describes repository independent configuration options for ORT.
+
+### Curating metadata of packages
+
+In order to scan the source code of dependencies, ORT relies on the metadata provided by those packages. Often the
+metadata contains information how to locate the source code, but this is not always the case. Often the metadata of
+packages has no VCS information, points to outdated repositories, or the repositories are not correctly tagged. Because
+this information can not always be fixed in remote packages ORT provides a mechanism to curate metadata of packages.
+
+These curations can be configured in a YAML file that has to be passed to the `analyzer`. The data from the curations
```

Comment:
I'm not sure whether we should be specific to the format here. Maybe just say "configuration file"? Or just "file" as you already say "configured" directly before.